components/org.wso2.carbon.identity.api.server.script.library/org.wso2.carbon.identity.api.server.script.library.v1/src/main/java/org/wso2/carbon/identity/api/server/script/library/v1/core/ServerScriptLibrariesService.java

```diff
@@ -0,0 +1,438 @@
+        } catch (IOException e) {
+            log.error("Error occurred while reading contentInputStream: " + e);
+            throw handleScriptLibraryClientError(Constants.ErrorMessage.ERROR_CODE_ERROR_UPDATING_SCRIPT_LIBRARY,
+                    Response.Status.INTERNAL_SERVER_ERROR);
+        }
+        if (isScriptLibraryAvailable(scriptLibraryName)) {
+            FunctionLibrary functionLibrary = createScriptLibraryPut(scriptLibraryName, scriptLibraryPUTRequest);
+            try {
+                ScriptLibraryServiceHolder.getScriptLibraryManagementService()
+                        .updateFunctionLibrary(scriptLibraryName, functionLibrary,
+                                ContextLoader.getTenantDomainFromContext());
+            } catch (FunctionLibraryManagementException e) {
+                throw handleScriptLibraryError(e, Constants.ErrorMessage.ERROR_CODE_ERROR_UPDATING_SCRIPT_LIBRARY);
+            }
+            return createScriptLibraryResponse(functionLibrary);
+        } else {
+            throw handleScriptLibraryClientError(Constants.ErrorMessage.ERROR_SCRIPT_LIBRARY_NOT_FOUND,
+                    Response.Status.NOT_FOUND, scriptLibraryName, ContextLoader.getTenantDomainFromContext());
+        }
+
+    }
+
+    /**
+     * Get the script library content.
+     *
+     * @param scriptLibraryName Name of the script library
+     * @return Script library content
+     */
+    public String getScriptLibraryContentByName(String scriptLibraryName) {
+
+        if (isScriptLibraryAvailable(scriptLibraryName)) {
+            try {
+                FunctionLibrary functionLibrary = ScriptLibraryServiceHolder.getScriptLibraryManagementService()
+                        .getFunctionLibrary(scriptLibraryName,
+                                ContextLoader.getTenantDomainFromContext());
+                return functionLibrary.getFunctionLibraryScript();
+            } catch (FunctionLibraryManagementException e) {
+                throw handleScriptLibraryError(e, Constants.ErrorMessage.ERROR_CODE_ERROR_RETRIEVING_SCRIPT_LIBRARY);
+            }
+        } else {
+            throw handleScriptLibraryClientError(Constants.ErrorMessage.ERROR_SCRIPT_LIBRARY_NOT_FOUND,
+                    Response.Status.NOT_FOUND, scriptLibraryName, ContextLoader.getTenantDomainFromContext());
+        }
+    }
+
+    /**
+     * Delete a script library.
+     *
+     * @param scriptLibraryId script library resource ID.
+     */
+    public void deleteScriptLibrary(String scriptLibraryId) {
+
+        if (isScriptLibraryAvailable(scriptLibraryId)) {
+            try {
+                ScriptLibraryServiceHolder.getScriptLibraryManagementService()
+                        .deleteFunctionLibrary(scriptLibraryId, ContextLoader.getTenantDomainFromContext());
+            } catch (FunctionLibraryManagementException e) {
+                throw handleScriptLibraryError(e, Constants.ErrorMessage.ERROR_CODE_ERROR_DELETING_SCRIPT_LIBRARY
```

Comment:
cant we merge line 252 and 253

Reply:
[fixed commit](https://github.com/wso2/identity-api-server/pull/81/commits/2800691d849b18ddbdf4fb3acff4c9f75b8e2795)